package/setup.py

```diff
@@ -267,6 +268,16 @@ def using_clang():
         is_clang = False
     return is_clang
 
+def using_msvc():
+    """Will we be using an MSVC compiler?"""
+    # how portable is this?
```

Comment:
I suppose checking if you're on Windows platform here/somewhere is fairly cheap as an extra guard, though not sure it would really help to have that extra filter.

Reply:
👍